concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-120	120	a short fur coat		
+RonaTas-2011-431-121	121	grist, groats, soft hail		
+RonaTas-2011-431-122	122	noon, south	12	MIDDAY
+RonaTas-2011-431-123	123	nut; walnut, Juglans	2042	WALNUT
+RonaTas-2011-431-124	124	pig, Sus domestica	1337	PIG
+RonaTas-2011-431-125	125	to lean, topple over, stream down	3619	LEAN
+RonaTas-2011-431-126	126	to squeeze, to thrust into	1414	SQUEEZE
+RonaTas-2011-431-127	127	church (building and organisation)	1349	CHURCH
+RonaTas-2011-431-128	128	wedge		
+RonaTas-2011-431-129	129	plough	2154	PLOUGH (INSTRUMENT)
+RonaTas-2011-431-130	130	to allow, permit, concede; yield, give way	3238	ALLOW
+RonaTas-2011-431-131	131	to propitiate		
+RonaTas-2011-431-132	132	community work in agriculture, a kind of corvée		
+RonaTas-2011-431-133	133	to arrive, to reach, get to	1387	ARRIVE
+RonaTas-2011-431-134	134	merit		
+RonaTas-2011-431-135	135	morals, morality		
+RonaTas-2011-431-136	136	to tire, lose vitality, slacken, relax	168	REST
+RonaTas-2011-431-137	137	strength, power	2526	POWER
+RonaTas-2011-431-138	138	oath	1712	OATH
+RonaTas-2011-431-139	139	reason, mind	1562	MIND
+RonaTas-2011-431-140	140	river, name of a river	666	RIVER
+RonaTas-2011-431-141	141	pelican, Pelicanus	472	PELICAN
+RonaTas-2011-431-142	142	polecat, Putorius foetidus		
+RonaTas-2011-431-143	143	matchmaker		
+RonaTas-2011-431-144	144	gleaner mouse, Mus spicilegus		
+RonaTas-2011-431-145	145	to abuse, calumniate	1953	ABUSE
+RonaTas-2011-431-146	146	drag-net, a kind of fishing net		
+RonaTas-2011-431-147	147	suspicion, mistrust	3197	SUSPICION
+RonaTas-2011-431-148	148	wool	964	WOOL
+RonaTas-2011-431-149	149	cotton	1850	COTTON
+RonaTas-2011-431-150	150	to increase, to put on weight, to grow stronger	1041	INCREASE
+RonaTas-2011-431-151	151	poor, frail, feeble	1674	POOR
+RonaTas-2011-431-152	152	to produce, build, fabricate	2886	PRODUCE
+RonaTas-2011-431-153	153	mourning, bereavement	3673	MOURNING
+RonaTas-2011-431-154	154	bulrush, Typha latifolia / angustifolia; matting made of bulrushes		
+RonaTas-2011-431-155	155	weak, feeble	1601	WEAK
+RonaTas-2011-431-156	156	rein		
+RonaTas-2011-431-157	157	borderland, hedge		
+RonaTas-2011-431-158	158	lawn,grass-(plot)	606	GRASS
+RonaTas-2011-431-159	159	child	2099	CHILD
+RonaTas-2011-431-160	160	candle	1273	CANDLE
+RonaTas-2011-431-161	161	hornbeam, Carpinus betulus		
+RonaTas-2011-431-162	162	murder, killer	845	MURDER
+RonaTas-2011-431-163	163	weed	3288	WEED
+RonaTas-2011-431-164	164	stomach	819	STOMACH
+RonaTas-2011-431-165	165	to confess (sins)		
+RonaTas-2011-431-166	166	cudweed, Gnaphalium; Edelweiss		
+RonaTas-2011-431-167	167	pearl	3325	PEARL
+RonaTas-2011-431-168	168	to torture, to make suffer	1952	TORTURE
+RonaTas-2011-431-169	169	to conquer (somebody, something), triumph (over somebody, something), gain victory	3849	TRIUMPH
+RonaTas-2011-431-170	170	to catch fire, be kindled, to be ignited		
+RonaTas-2011-431-171	171	to knead, pug	274	KNEAD
+RonaTas-2011-431-172	172	to assemble, gather	7	GATHER
```

Comment:
I'd change the mapping to `3638 ASSEMBLE (GET TOGETHER)`

Reply:
A similar gloss is mapped to GATHER in [Huang-1992-1820-1402](https://concepticon.clld.org/values/Huang-1992-1820-1402). @martino-vic Can you disambiguate the gloss? Otherwise, I'd unmap.

Reply:
The original word \<gyűlik\> mostly means "to gather". I think the other meaning is again there to explain the semantic processes underlying the etymology

Reply:
Resolved in 2c64368e0fcb7cc87964075b2696cbf49d3f68ba